CRVResponse/fcl/prolog_v11.fcl

```diff
@@ -51,19 +51,30 @@ BEGIN_PROLOG
-                                   39400,                                //U
-                                   39400,39400,39400,39400,              //D1...4
-                                   0,0,39400,39400]                      //C1...4
+                                //all measurements were done at 1m away from SiPM with a reference date of 12/2023
+                                //CRV-Top (1.8mm) modules: 39794 - based on comparison between Wideband measurements and fined-tuned simulation
+                                //                         (all counters come from scintillator batch 1)
+                                //CRV-Top (1.4mm) modules: 30442 - based on comparions between UVA measurements of 1.4mm and 1.8mm CRV-Top modules (adjusted for age)
+                                //                         (most counters come from scintillator batch 1)
+                                //                         assuming 3% aging per year
+                                //                         T (1.4mm) measured 2/2022 at 44.5 PEs, 1.83 years to 12/2023, aging factor 0.946, --> 42.1 PEs
+                                //                         T (1.8mm) measured 8/2022 at 57.3 PEs, 1.33 years to 12/2023, aging factor 0.960, --> 55.0 PEs
+                                //                         --> photon yield ratio: 0.765
+                                //CRV-TS modules:          30442 - same as CRV-Top (1.4mm), because all counters come from scintillator batch 1
+                                //CRV-R19-T module:        30442 - same as CRV-Top (1.4mm), because all counters come from scintillator batch 1
+                                //all other modules:       28573 - counters come from scintillator batch 2, with a photon yield ratio of 1.07/1.14
+                                //CRV-U, CRV-TS-Ext:       23811 - reference CRV-Top (1.8mm) was measured at 3.0V overvoltage,
+                                //                         but CRV-U/CRV-TS-Ext will run at 2.5V --> scale to 2.5/3.0
+      scintillationYields       : [28573,28573,30442,28573,28573,28573,  //R1...6
+                                   28573,28573,28573,                    //L1...3
+                                   30442,30442,39794,39794,30442,        //T1...5
+                                   23811,23811,                          //E1,E2
+                                   23811,                                //U
+                                   28573,28573,28573,28573,              //D1...4
+                                   0,0,28573,28573]                      //C1...4
 
-      photonYieldScaleFactor                : 0.42  //scale factor applied to photon yields to mimic aging
-                                                    //0.42 results in 28.6 PE/SiPM @ 1 m away from SiPM (expected in 2025)
+      photonYieldScaleFactor                : 0.91  //scale factor applied to photon yields to mimic aging
```

Comment:
Are we aiming for Run-1?
Should we consider the Run-2 scenario to be conservative?

Reply:
Not sure. I think it was also for Run I in the previous version.

Reply:
That's a big change. We have to be prepared for a large impact on the noise rates and efficiencies.

Reply:
The previous photon yields were at 39400*0.42=16548 for all channels. The current photon yields at the CRV-U and CRV-TS-Ext are 23811*0.91=21668 (all other channels are higher). Perhaps we should move it to e.g. 2026 or later? Or apply a faster aging? I'm using 3% per year.

Reply:
I think we should use the best knowledge we currently have.
The aging rate of 3%/year sounds reasonable.
We agreed that we should use the Dec 2026 as the start date of Run-1.
You can pick the midpoint of Run-1, but that's a tiny detail and not well-defined.
Are we sure that Run-1 will be 7 months long? 

So, I would just keep what you have for now.
Later, we might need to produce dedicated studies varying photonYieldScaleFactor to study an impact on rates and efficiency.